diff --git a/platform/radio/wifi/wfx_fmac_driver/component/wfx_bus_spi.slcc b/platform/radio/wifi/wfx_fmac_driver/component/wfx_bus_spi.slcc
index 626531ec1..7a1aa73f2 100644
--- a/platform/radio/wifi/wfx_fmac_driver/component/wfx_bus_spi.slcc
+++ b/platform/radio/wifi/wfx_fmac_driver/component/wfx_bus_spi.slcc
@@ -22,8 +22,13 @@ config_file:
   - override:
       component: "wfx_bus_spi"
       file_id: "sl_wfx_host_bus_pinout"
-    path: "config/brd2204a/spi/sl_wfx_host_bus_pinout.h"
+    path: "config/brd2204x/spi/sl_wfx_host_bus_pinout.h"
     condition: [brd2204a]
+  - override:
+      component: "wfx_bus_spi"
+      file_id: "sl_wfx_host_bus_pinout"
+    path: "config/brd2204x/spi/sl_wfx_host_bus_pinout.h"
+    condition: [brd2204c]
   - override:
       component: "wfx_bus_spi"
       file_id: "sl_wfx_host_bus_pinout"
diff --git a/platform/radio/wifi/wfx_fmac_driver/component/wfx_fmac_driver.slcc b/platform/radio/wifi/wfx_fmac_driver/component/wfx_fmac_driver.slcc
index c0c56d5cf..0557fd616 100644
--- a/platform/radio/wifi/wfx_fmac_driver/component/wfx_fmac_driver.slcc
+++ b/platform/radio/wifi/wfx_fmac_driver/component/wfx_fmac_driver.slcc
@@ -36,6 +36,8 @@ requires:
 recommends:
   - id: "wfx_bus_spi"
     condition: [brd2204a]
+  - id: "wfx_bus_spi"
+    condition: [brd2204c]
 root_path: "platform/radio/wifi/wfx_fmac_driver"
 include:
   - path: "./"
@@ -91,8 +93,13 @@ config_file:
   - override:
       component: "wfx_fmac_driver"
       file_id: "sl_wfx_host_pinout"
-    path: "config/brd2204a/sl_wfx_host_pinout.h"
+    path: "config/brd2204x/sl_wfx_host_pinout.h"
     condition: [brd2204a]
+  - override:
+      component: "wfx_fmac_driver"
+      file_id: "sl_wfx_host_pinout"
+    path: "config/brd2204x/sl_wfx_host_pinout.h"
+    condition: [brd2204c]
   - override:
       component: "wfx_fmac_driver"
       file_id: "sl_wfx_host_pinout"
diff --git a/platform/radio/wifi/wfx_fmac_driver/config/brd2204a/sl_wfx_host_pinout.h b/platform/radio/wifi/wfx_fmac_driver/config/brd2204x/sl_wfx_host_pinout.h
similarity index 100%
rename from platform/radio/wifi/wfx_fmac_driver/config/brd2204a/sl_wfx_host_pinout.h
rename to platform/radio/wifi/wfx_fmac_driver/config/brd2204x/sl_wfx_host_pinout.h
diff --git a/platform/radio/wifi/wfx_fmac_driver/config/brd2204a/spi/sl_wfx_host_bus_pinout.h b/platform/radio/wifi/wfx_fmac_driver/config/brd2204x/spi/sl_wfx_host_bus_pinout.h
similarity index 100%
rename from platform/radio/wifi/wfx_fmac_driver/config/brd2204a/spi/sl_wfx_host_bus_pinout.h
rename to platform/radio/wifi/wfx_fmac_driver/config/brd2204x/spi/sl_wfx_host_bus_pinout.h
